.ado/templates/android-build-office.yml

```diff
@@ -29,6 +29,12 @@ steps:
     displayName: npm install
     inputs:
       script: npm install
+
+  - task: CmdLine@2
```

Comment:
We don't need to do this for Apple builds as well?

Reply:
Since the react-native npm package contains built android binaries, its the android build that actually publishes the react-native package.